Tests assume a CET timezone, but do not respect the "TZ" variable.

This patch is a "squashed" version of these upstream commits:
<https://cgit.freedesktop.org/libreoffice/libvisio/commit/?id=a97d30ad693374deab404ec31fe00665882cc949>
<https://cgit.freedesktop.org/libreoffice/libvisio/commit/?id=c933df45a873e566c6ce4e5de5f829e64eb892f5>

diff --git a/src/test/Makefile.am b/src/test/Makefile.am
index 59d3419..23049b5 100644
--- a/src/test/Makefile.am
+++ b/src/test/Makefile.am
@@ -29,4 +29,7 @@ EXTRA_DIST = \
 	     data/dwg.vsdx \
 	     $(test_SOURCES)
 
+# ImportTest::testVsdMetadataTitleUtf8 checks formatted date string
+AM_TESTS_ENVIRONMENT = TZ=UTC; export TZ;
+
 TESTS = test
diff --git a/src/test/importtest.cpp b/src/test/importtest.cpp
index e05b3c1..32fb185 100644
--- a/src/test/importtest.cpp
+++ b/src/test/importtest.cpp
@@ -242,8 +242,8 @@ void ImportTest::testVsdMetadataTitleUtf8()
   // Test the case when the string is UTF-8 encoded already in the file.
   assertXPath(m_doc, "/document/setDocumentMetaData", "title", "mytitle\xC3\xA9\xC3\xA1\xC5\x91\xC5\xB1");
   // Test <dcterms:created> and <dcterms:modified>.
-  assertXPath(m_doc, "/document/setDocumentMetaData", "creation-date", "2014-11-26T09:24:56Z");
-  assertXPath(m_doc, "/document/setDocumentMetaData", "date", "2014-11-26T09:24:56Z");
+  assertXPath(m_doc, "/document/setDocumentMetaData", "creation-date", "2014-11-26T08:24:56Z");
+  assertXPath(m_doc, "/document/setDocumentMetaData", "date", "2014-11-26T08:24:56Z");
 }
 
 void ImportTest::testVsdUserDefinedMetadata()
-- 
2.1.4
